src/sites/amsterdam/components/contact/Contact.tsx

```diff
@@ -0,0 +1,64 @@
+import { Breadcrumb, Column, Fieldset, Grid, Heading, Link, Paragraph, Radio } from '@amsterdam/design-system-react'
+import { siteUrl } from '../../constants'
+import FormNavigation from '../shared/FormNavigation'
+import { FormEvent } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+export const Contact = () => {
+  const navigate = useNavigate()
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    const formData = new FormData(event.currentTarget)
+    const soort = formData.get('soort')
+
+    if (typeof soort === 'string') {
+      navigate(`${siteUrl}contact/${soort}`)
+    }
+  }
+
+  return (
+    <Grid paddingBottom="medium">
+      <Grid.Cell span={{ narrow: 4, medium: 5, wide: 8 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+        <Breadcrumb>
+          <Breadcrumb.Item href={siteUrl}>Home</Breadcrumb.Item>
+          <Breadcrumb.Item href="#">Contact</Breadcrumb.Item>
+        </Breadcrumb>
+        <Column className="ams-mb--md" gap="small">
```

Comment:
This `Column` can be removed. It has only one child, so the gap doesn’t appear, and we don’t need the bottom margin. There may be more occurrences.